text_extensions_for_pandas/io/bert.py

```diff
@@ -216,7 +216,7 @@ def align_bert_tokens_to_corpus_tokens(
         [["corpus_token", "ent_type"]]
             .rename(columns={"corpus_token": "span"})
     )
-    cons_df["span"] = TokenSpanArray.align_to_tokens(
+    cons_df["token_span"] = TokenSpanArray.align_to_tokens(
```

Comment:
@frreiss before this change it is replacing the existing "span" column, could you just verify that is what we want or should we do this to make a new "token_span" column?

Reply:
this is also the cause of the current test failure

Reply:
Returning both a token-based span and the original makes sense. Would you mind updating the description of the returned DataFrame's schema in the docstring above to match? If I'm reading this code correctly, the schema should be `[span, token_span, ent_type]`, where `span` is the original span from the input and `token_span` is a token-based span aligned to the BERT tokens.

Reply:
On second thought, adding an additional column "token_span" makes the notebook a bit more confusing and later functions are set to work with a "span" column, so it would need to be changed too.

I revert my changes so that the "span" column is replaced by a TokenSpanArray, as before, and updated the docs. I think it's clearer this way, wdyt?